### ISSUE TITLE
Href links broken for Attachments in Shortcodes

### ISSUE THREAD

**Issue by llcooljayr** [vjeantet/hugo-theme-docdock#18]
I ran the sample code as a test project and the links would not open when I click on any of the attachments:

The url that is open is this: http://localhost:1313/shortcodes%5Cattachments.files/BachGavotteShort.mp3

The correct url should be http://localhost:1313/shortcodes/attachments.files/BachGavotteShort.mp3

Not sure why there is an html encoded '%5C' in the url which the browser does not understand.

![error](https://user-images.githubusercontent.com/3460948/27155613-76e5dae4-510e-11e7-94f3-4214dceee203.png)

**Comment by vjeantet** [comment]
Ok thanks I'll fix this thanks for the report

**Comment by vjeantet** [comment]
Hello @llcooljayr, i can not reproduce your issue.. neither on local server or online.

see : http://docdock.netlify.com/shortcodes/attachments/

Can you provide me a link to your repo ?

**Comment by llcooljayr** [comment]
Hi @vjeantet,

Thanks for the reply. I have been testing on my local server. Let me test again and get back to you.

**Comment by altinnadmin** [comment]
Hi @vjeantet,

I can reproduce this problem on Windows.
The problem is that $.Page.File.Dir on Windows returns backslash (%5C) instead of slash.
If you replace \ with / before building the links in the [attachments shortcode](https://github.com/vjeantet/hugo-theme-docdock/blob/master/layouts/shortcodes/attachments.html), it will work.

Something like this:
```golang
{{ $fileDir := replace $.Page.File.Dir "\\" "/" }}
```
...
```
<a href="{{ printf "%s/%s%s/%s" $.Site.BaseURL $fileDir ($.Scratch.Get "filesName") .Name }}" >
	{{.Name}}
</a>
```

I will submit PRs with some other bugfixes/enhancements after porting our site to the latest version.

Your work is much appreciated!! :)

**Comment by vjeantet** [comment]
Thanks @altinnadmin , we wait for your PR

**Comment by altinnadmin** [comment]
@vjeantet I created a PR containing the fix in 833c1a9. Going forward I'll create one PR for each fix/enhancement.

**Comment by vjeantet** [comment]
Hello @llcooljayr can you update and confirm resolution with @altinnadmin 's PR
